templates/user/profile.html

```diff
@@ -55,8 +55,8 @@ <h1 name=username>{{ user.username or 'Insert a username' }}</h1>
     <p><span class=newtext>&nbsp;</span><span class=newtext>&nbsp;</span><span class=newtext>&nbsp;</span></p>
     <a href={{ url_for('article_create') }} class=read>Write this article</a>
     <form>
-      <input name="published" id="published" type="checkbox">
-      <label for="published" title="draft or publish">published</label>
+      <input name=published type=checkbox>
+      <label for='published' title=draft or publish>published</label>
```

Comment:
no need of quotes, even single quotes.
The `for` links to an `id`. You just deleted the id…